src/index.js

```diff
@@ -36,20 +36,21 @@ const counterSchema = new mongoose.Schema({
   count: { type: Number, default: 0 },
 });
 
-counterSchema.index(
-  {
-    field: 1,
-    groupingField: 1,
-    model: 1,
-  },
-  {
-    unique: true,
-  }
-);
+// Deprecated after upgrading to Mongoose5.
```

Comment:
It  made the error not get triggered, but its aint a solution ;]. Its not deprecated as such.  Have looked back at the 5.0.2 doc. 
```
animalSchema.index({ name: 1, type: -1 }); 
```
should be still valid ^^

ps. guess you are working on it but maybe lets mark the PR as WIP? ;]